ui/components/multichain/import-tokens-modal/index.scss

```diff
@@ -1,6 +1,32 @@
+.import-tokens-content-scrollable {
```

Comment:
With these components we try to have one parent CSS className, and everything else name off of that, so something like:

```
.import-tokens-modal {
    &__tabs { /* aka .import-tokens-modal__tabs */
 
    }

    &__search-list { /* .import-tokens-modal__search-list */

    }
}
```

Your changes include a few sibling-like classNames ; can we consolidate?